operator/v1/types_ingress.go

```diff
@@ -60,6 +60,16 @@ type IngressControllerSpec struct {
 	// +optional
 	Domain string `json:"domain,omitempty"`
 
+	// httpErrorCodePages specifies a configmap with custom error pages.
+	// The administrator must create this configmap in the openshift-config namespace.
+	// This configmap should have keys in the format "error-page-<error code>.http",
+	// where <error code> is an HTTP error code.
+	// For example, "error-page-503.http" defines an error page for HTTP 503 responses.
+	// Currently only error pages for 503 and 404 responses can be customized.
+	// Each value in the configmap should be the full response, including HTTP headers.
```

Comment:
how big a sample format?  it's not obvious to me what I would place in this configmap value.

Reply:
A sample format will be pretty big. It will be like this https://github.com/openshift/enhancements/pull/680/commits/e77e05d49f3f1f9b203c6d5f24fc9a1006eb666f#diff-37fc66033a40ad5f0980172e2fea31e86c0cb46a3a83eaee9dac44f221f2188bR82  I will add link referring correct http response

Reply:
That wouldn't be very useful; the godoc loses all formatting, so an example would be illegible.

Reply:
https://github.com/openshift/router/blob/master/images/router/haproxy/conf/error-page-503.http (the default HTTP 503 error page) is a representative sample, and it's 3.34 KiB.

Reply:
> https://github.com/openshift/router/blob/master/images/router/haproxy/conf/error-page-503.http (the default HTTP 503 error page) is a representative sample, and it's 3.34 KiB.

Can you create a stable link to this and add it here please?  I suspect many  users will have similar questions/requests.

Reply:
https://raw.githubusercontent.com/openshift/router/fadab45747a9b30cc3f0a4b41ad2871f95827a93/images/router/haproxy/conf/error-page-503.http